tests/test_property.py

```diff
@@ -1734,3 +1734,227 @@ def test_slice_for_box(example_model_with_properties):
 
     # Assert
     assert_array_almost_equal(expected, myslice)
+
+
+def test_coarsening_length(example_fine_coarse_model):
+    # Arrange
+    model, coarse, fine, fc = example_fine_coarse_model
+    # Remove existing length parts
+    coarse_pc = rqp.GridPropertyCollection(grid = coarse)
+    for part in coarse_pc.parts():
+        if coarse_pc.citation_title_for_part(part) in ['DX', 'DY', 'DZ']:
+            coarse_pc.remove_part_from_dict(part)
+    # Check on number of properties
+    numc = len(coarse_pc.parts())
+    fine_pc = rqp.GridPropertyCollection(grid = fine)
+    numf = len(fine_pc.parts())
+
+    # Act
+    coarse_pc.extend_imported_list_copying_properties_from_other_grid_collection(other = fine_pc, coarsening = fc)
+    coarse_pc.write_hdf5_for_imported_list()
+    coarse_pc.create_xml_for_imported_list_and_add_parts_to_model()
+
+    # Assert
+    assert len(coarse_pc.parts()) == numc + numf
+    all_tens = np.zeros(shape = (3, 5, 5)) + 10
```

Comment:
You might like to use the numpy full() function instead of this construct:
all_tens = np.full(shape = (3, 5, 6), fill_value = 10)

Reply:
PS. With the full() version, it would be slightly better to setup a float array, so fill_value = 10.0 or dtype = float